src/app/pages/send/send-crypto-asset-form/form/stacks/stacks-send-form-confirmation.tsx

```diff
@@ -44,6 +47,25 @@ export function StacksSendFormConfirmation() {
 
   useRouteHeader(<ModalHeader hideActions defaultClose defaultGoBack title="Review" />);
 
+  const feeWarningTooltip = showFeeChangeWarning ? (
+    <Tooltip
+      label={
+        'The wallet noticed you are using a nonce for this transaction which is already pending. The fee has been increased so that it is exactly high enough to replace the pending transaction with the same nonce.'
```

Comment:
```suggestion
        'You are using a nonce for this transaction that is already pending. The fee has been increased so that it is exactly high enough to replace the pending transaction with the same nonce.'
```

"the wallet noticed" reads badly i think